#! /bin/sh /usr/share/dpatch/dpatch-run
## ../makefile-CC.diff.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use CC variable

@DPATCH@

 wmitime/Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/wmitime/Makefile b/wmitime/Makefile
index 1b1185d..14308a5 100644
--- a/wmitime/Makefile
+++ b/wmitime/Makefile
@@ -2,17 +2,17 @@
 LIBDIR = -L/usr/X11R6/lib
 LIBS   = -lXpm -lXext -lX11 -lm
 FLAGS = -O2
+CC     = gcc
 OBJS =	wmitime.o \
 		../wmgeneral/wmgeneral.o \
 		../wmgeneral/misc.o \
 		../wmgeneral/list.o
 
-
 .c.o:
-	cc -I/usr/X11R6/share/include $(FLAGS) -D$(LANG) -c -Wall $< -o $*.o
+	$(CC) -I/usr/X11R6/share/include $(FLAGS) -c -Wall $< -o $*.o
 
 wmtime: $(OBJS)
-	cc $(FLAGS) -D$(LANG) -o wmitime $^ -lXext $(LIBDIR) $(LIBS)
+	$(CC) $(FLAGS) -o wmitime $^ -lXext $(LIBDIR) $(LIBS)
 
 all:: wmtime
 
